index.html

```diff
@@ -305,6 +305,53 @@ <h3 class="title-about">
           </ul>
         </section>
       </section>
+
+      <!-- contact section -->
+      <footer class="footer">
+        <h2 class="contact-title title">Contact me</h2>
+        <article class="contact-info">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </article>
+        <form
+          action="https://formspree.io/f/xknydra"
+          method="post"
+          class="input form"
+        >
```

Comment:
- [ ] As stated the project requirement Kindly make sure that the form submitted successfully, because when I try to submit the form I got the below error, kindly check that you used the correct `form_id` when you integrate with [formspree service](https://formspree.io/html).
- Your Form when Submitted : 

![Capture](https://user-images.githubusercontent.com/93598189/220428900-e6dd3ede-f8c5-41e8-9d1f-22545b4c604a.PNG)